From 5fd3b42db673bb19671ac4835ba924ba46694960 Mon Sep 17 00:00:00 2001
From: Bo Shen <voice.shen@atmel.com>
Date: Fri, 6 Mar 2015 11:32:58 +0800
Subject: [PATCH] board: at91sam9261ek: remove 266MHz support

There is no clock configuration for 266MHz of at91sam9261ek,
remove it.

Signed-off-by: Bo Shen <voice.shen@atmel.com>
Acked-by: Nicolas Ferre <nicolas.ferre@atmel.com>
---
 board/at91sam9261ek/Config.in.board | 1 -
 1 file changed, 1 deletion(-)

diff --git a/board/at91sam9261ek/Config.in.board b/board/at91sam9261ek/Config.in.board
index 85d4188..3b17a7e 100644
--- a/board/at91sam9261ek/Config.in.board
+++ b/board/at91sam9261ek/Config.in.board
@@ -5,7 +5,6 @@ config	CONFIG_AT91SAM9261EK
 	select ALLOW_DATAFLASH
 	select ALLOW_NANDFLASH
 	select ALLOW_CPU_CLK_200MHZ
-	select ALLOW_CPU_CLK_266MHZ
 	select ALLOW_CRYSTAL_18_432MHZ
 	select ALLOW_BOOT_FROM_DATAFLASH_CS0
 	select ALLOW_BOOT_FROM_DATAFLASH_CS3
-- 
1.9.1

